packages/keys/src/key-pair.ts

```diff
@@ -9,3 +11,15 @@ export async function generateKeyPair(): Promise<CryptoKeyPair> {
     );
     return keyPair as CryptoKeyPair;
 }
+
+export async function createKeyPairFromBytes(bytes: Uint8Array, extractable?: boolean): Promise<CryptoKeyPair> {
+    if (bytes.length !== 64) {
```

Comment:
I've been used to writing `byteLength`. Is that literally just an alias for `length` or is there something more subtle?

I think it happens to be ‘the same’ particularly for `Uint8Array` (ie. the number of elements in the array happens to also be the number of bytes) but it's not true for multibyte integers.

```ts
new Uint16Array([1]).byteLength; // 2
new Uint16Array([1]).length; // 1
```

Reply:
I'll change it to `byteLength` then for consistency.